http-builder-ng-core/src/main/java/groovyx/net/http/UriBuilder.java

```diff
@@ -437,14 +516,14 @@ public String getHost() {
             return host;
         }
 
-        private volatile GString path;
+        private volatile String path;
 
-        public UriBuilder setPath(GString val) {
+        public UriBuilder setPath(String val) {
             path = val;
             return this;
         }
 
-        public GString getPath() {
+        public String getPath() {
```

Comment:
@dwclark do you remember why this was a `GStrring` rather than just a `String`?

Reply:
Yes, it's for logging and diagnostics support. Simple example, suppose I have a basic REST endpoint that gets people by id with a path like this:

`"/person/${id}"`

Since we take GString the content remains uninterpolated until the last possible moment. This makes it easy to log or gather metrics on the uninterpolated value. If we force it to be a string then Groovy will do the substitution of the id and the logging/metrics would not see a single endpoint but rather lots of stuff like this:

```
/person/100
/person/576
/person/1000000
/person134
```
and so on. This leads to confusion in trying to do things like add metrics with DropWizard. You end up with millions of metrics all with a single value.